main.py

```diff
@@ -0,0 +1,80 @@
+import requests
+import json
+from bs4 import BeautifulSoup
+import pandas as pd
+
+
+def get_team_id_season_id(url: str) -> list:
+    try:
+        response = requests.get(url)
+    except requests.exceptions.Timeout:
+        print("Request Timed Out.. Please Check Internet Connection Or Use A Valid Proxy..")
+        return ["-1", "-1"]
+    except requests.exceptions.TooManyRedirects:
+        print("Url Is Not Valid.. Please check the demo url in github and try url like that..")
+        return ["-1", "-1"]
+    except requests.exceptions.RequestException as e:
+        print("Error Occurred While Fetching Data ", e)
+        return ["-1", "-1"]
+
+    soup = BeautifulSoup(response.text, "html.parser")
```

Comment:
I would also add a try catch over this block that catches everything below to catch changes in the website that will break this. 

In that except, use: traceback.print_exc() to print out the traceback, and then printout a user friendly message below it explaining a problem happened when parsing the website